aiohttp/streams.py

```diff
@@ -112,8 +114,10 @@ def __repr__(self):
             info.append('%d bytes' % self._size)
         if self._eof:
             info.append('eof')
-        if self._limit != DEFAULT_LIMIT:
-            info.append('l=%d' % self._limit)
+        if self._low_water != DEFAULT_LIMIT:
+            info.append('low=%d' % self._low_water)
+        if self._high_water != DEFAULT_LIMIT * 2:
```

Comment:
May be always show low/high watter in repr? Otherwise it's not oblivious: it has no limits or just some hidden default ones?

Reply:
Done